apps/nextjs-website/src/lib/strapi/fetchFromStrapi.ts

```diff
@@ -28,6 +28,7 @@ export const fetchFromStrapi = <A, O, I>(
             () =>
               fetchFun(`${strapiEndpoint}/api/${path}/?${populate}`, {
                 method: 'GET',
+                cache: 'no-store',
```

Comment:
I think you should remove this change from this PR.
This option is useful for local development, but useless in production and anyhow is not part of this task